sp800-63b/sec6_lifecycle.md

```diff
@@ -33,30 +33,21 @@ Enrollment and binding MAY be broken up into a number of separate physical encou
-
-***Consider what proofing information the CSP is allowed to maintain. Privacy impact here?***
+If the account has only one authentication factor bound to it (which is possible only at IAL 1/AAL 1), an additional authenticator may be added, and if it is a different factor (e.g., *something you have* added when authenticated with *something you know*), the subscriber MAY request that the account be upgraded to AAL 2 (but still at IAL 1). Once this has been done, it SHALL no longer be possible for the subscriber to use single-factor authentication.
 
+If the subscriber loses one of the two authentication factors and has been identity proofed at IAL 2 or 3, they SHALL repeat the identity proofing process, using the remaining authenticator and SHOULD verify knowledge of some information collected during the proofing process to bind to the existing identity. Reestablishment of authentication factors at IAL 3 SHALL be done in person and SHALL verify the biometric collected during the proofing process.
 
 #### 6.1.3. Binding Identity to a Subscriber Provided Authenticator
-In some instances, a claimant may already possess authenticators at a suitable AAL without having been proofed at the equivalent IAL. For example, a user may have a two-factor authenticator from a social network provider, considered AAL2 and IAL1, and would like to use those credentials at a relying party that requires IAL2.
 
-{::comment}
-
-**I think we are making this too hard.  And not sure this is correct if we even want this as increasing IAL should still require address confirmation.  I'm thinking we require address confirmation and then let the binding of an existing authenticator to occur, so the OR below turns into an AND.**
-
-{:/comment}
+In some instances, a claimant may already possess authenticators at a suitable AAL without having been proofed at the equivalent IAL. For example, a user may have a two-factor authenticator from a social network provider, considered AAL2 and IAL1, and would like to use those credentials at a relying party that requires IAL2.
 
 The following requirements apply when a claimant choses to increase IAL in order to bind to a suitable authenticator they already have.
 
-1. The CSP MAY accept an existing authenticator at or above the desired IAL
-2. The CSP SHALL require the user to authenticate using their existing authenticator
+1. The CSP MAY accept existing one or more existing authenticators
+2. The CSP SHALL require the user to authenticate using their existing authenticator(s) at an AAL compatible with the desired IAL (as defined in [SP 800-63-3](sp800-63-3.html), Acceptable IAL and AAL Combinations table)
```

Comment:
Even if there are no other assertions being conveyed to the RP, the result of authentication is the assertion of an identifier that allows the RP to associate this session with previous authenticated sessions from the same user. That identifier needs an assurance level too, which is the AAL.